.github/workflows/cff_validation.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: validate citation.cff
+
+on:
+  push:
+    branches: ['*']
```

Comment:
I think we can probably just run this against `main` since we know what our branches are called.